datasets/wikicorpus/README.md

```diff
@@ -0,0 +1,219 @@
+---
+annotations_creators:
+  raw_ca: []
+  raw_en: []
+  raw_es: []
```

Comment:
could you fix these? There's a `no-annotation` option in the tagging app

Reply:
Done.